### DISCUSSION TITLE
Editor Component needs to be explicitly imported

### DISCUSSION THREAD

**Discussion by felix-dolderer** [primefaces/community#463] [PrimeVue]
The Editor Component only works if it is explicitly imported.
Usually components work fine with being auto-imported (all other components I used so far).

Reproduction repo: https://github.com/felix-dolderer/primevue-editor-issue

Further notes:
- Issue comes up regardless of using nuxt or plain vue
- Issue comes up regardless of including tailwind or not
- Issue comes up regardless of using bun or npm

**Comment by FlipWarthog** [comment]
Looks like the nuxt-primevue module explicitly excludes Editor and Chart from auto-import: https://github.com/primefaces/primevue-nuxt-module/blob/main/src/module.ts#L23

I'd bet it's because both components have third party deps, Quill and Chart.js.
